pubnub-core/pubnub-core-impl/src/main/kotlin/com/pubnub/internal/PubNubCore.kt

```diff
@@ -1315,6 +1315,7 @@ class PubNubCore internal constructor(
         includeCount: Boolean = false,
         includeCustom: Boolean = false,
         includeChannelDetails: PNChannelDetailsLevel? = null,
+        includeType: Boolean = false,
```

Comment:
What is a reason we want to expose `includeType` ?

Reply:
it's needed for ChatSDK, otherwise returned data will always have a type UNKNOWN

besides, it's present in the REST API and so far it wasn't implemented in the SDK

Reply:
I see. Thank's.